File: ClientApp/ClientApp/Models/Responses/ResetPasswordResponse.cs
using ClientApp.Models.ResponseErrors;
using Newtonsoft.Json;

namespace ClientApp.Models.Responses
{
    class ResetPasswordResponse : Response
    {
        private ResetPasswordResponse() { }
        [JsonProperty]
        public ResetPasswordErrors Errors { get; private set; }
        public override ResponsesCodes GetCode() => ResponsesCodes.ResetPasswordResponse;
    }
}

File: M_Trivia/M_Trivia/ResetPasswordResponse.cpp
#include "ResetPasswordResponse.hpp"
#include "json.hpp"




ResetPasswordResponse::ResetPasswordResponse(ResetPasswordResponseErrors* resetPasswordResponseErrors)
	: Response(resetPasswordResponseErrors	) {
}


ResponseCodes ResetPasswordResponse::getCode() const 
{
	return ResponseCodes::C_ResetPasswordResponse;
}

